boilerplate/app/components/Screen.tsx

```diff
@@ -194,7 +197,8 @@ function ScreenWithScrolling(props: ScreenProps) {
   useScrollToTop(ref)
 
   return (
-    <ScrollView
+    <KeyboardAwareScrollView
+      bottomOffset={DEFAULT_BOTTOM_OFFSET}
```

Comment:
I think this should come in from a prop and we can set this default if it makes sense, but right now this will just always be 50 correct?

Reply:
@frankcalise added new prop!

Reply:
@fpena Great thanks for doing that!